rulebook.tex

```diff
@@ -250,83 +278,103 @@ \section{Introduction}
-competing with a second team against the clock. In the RCLL,
-autonomous robot agents have to handle the logistics of materials
-through several (dynamic) stages to produce final goods to fulfill
-orders. The machines are specific MPS stations.
-
-The Logistics League's main challenge is a multistage production cycle
-of different product variants with self-crafted intermediate products
-and delivery of final products. This genuine goal will be rewarded
-considerably higher than partial fulfillment of the task. Autonomous
-robots transport small products between the processing
-machines. Machines are MPS stations which complete a particular
-refinement step like mounting a colored ring or top-most cap. If
-procurable this will lead to a new sub-assembly or final product
-(cap). Complete work orders require all related sub-assemblies of
-product variants (cf. \refsec{sec:production-complexities}).
-
-This work flow is controlled by a referee box broadcasting information
-via wifi (see \refsec{sec:referee-box}). The work flow itself is
-divided into three different phases: a setup
-phase (see \refsec{sec:setup-phase}), an exploration phase (see
-\refsec{sec:exploration-phase}) during which robots receive scores for
-correctly discovering and publishing the positions of the yet
-unknown different machines on the field. After this phase the referee
-box will announce all machine types and designations. In the following
-production phase (see~\refsec{sec:production-phase}) orders are
-announced by the referee box, which the robots must fulfill
+ Our aim is to provide a simplified Smart Factory environment, where
+ teams of autonomous robots must operate a flexible production plant without
+ human interference and against the clock.
+ The Logistics League's core task is to fulfill placed product orders by
+ executing multi-stage production cycles, before delivering the finished
+ products within the requested time frame.
+ This is achieved by transporting small pieces of material between assembly
+ machines (\ac{MPS} stations, see \refsec{sec:machines}),
+ which can be instructed to perform different refinement steps,
+ such as mounting a colored ring on a cylindrical base product.
+ Efficient handling of the material logistics and utilization of the available
+ machines are core requirements to succeed in the \ac{RCLL}.
+
+ This work flow is controlled by a \acf{refbox}~\cite{RCI-RefBox}%
+\footnote{Code and documentation available at
+\url{https://github.com/robocup-logistics/rcll-refbox}}
+broadcasting information via wifi (see \refsec{sec:referee-box}).
+It is a software system that runs on a
+system provided by the Organizing Committee and controls the overall game,
+monitors feedback from the robots, and awards points.
+The work flow itself is divided into phases:
+During a setup phase (see \refsec{sec:setup-phase})
+the field setup is prepared and the robots are started.
+Then the \ac{refbox} switches to the production phase, during which the
```

Comment:
Right now there is still the Exploration phase which I would also mention here, and I think having a reference to both of them would be appropriate here.

Reply:
True, done in https://github.com/robocup-logistics/rcll-rulebook/pull/43/commits/c377ee9b79abb2f0415c18a518e7a8132b5c8a74